src/editors/containers/VideoUploadEditor/index.scss

```diff
@@ -32,12 +34,22 @@
 
   input::placeholder {
     color: #454545;
+    // color: #5E35B1;
+    font-size: 14;
+    font-family: 'Inter';
+    font-weight: '500';
+    line-height: 24;
+    word-wrap: 'break-word';
   }
 
   button {
     border: none !important;
     background-color: #FFFFFF;
   }
+
+  .prompt-button {
+    background: rgba(239, 234, 247, 0.70);
+  }
 }
 
 .error-message {
```

Comment:
A lot of this file contains magic values. Can you instead use variables so that it picks up the correct theme?

You can see the variables here: https://paragon-openedx.netlify.app/foundations/css-utilities/
Find the matching variable for what you want. I think you might need to look at the edX theme, since I think figma designs generally use the edX theme.

Reply:
I have tried removing as much as I can :)